src/theme/components.tsx

```diff
@@ -209,12 +209,15 @@ export function ExternalLinkIcon({
   )
 }
 
-const ToolTipWrapper = styled.div<{ isCopyContractTooltip?: boolean }>`
+const TOOLTIP_WIDTH = 60
+
+const ToolTipWrapper = styled.div<{ isCopyContractTooltip?: boolean; tooltipX?: number }>`
```

Comment:
is this the x coordinate on the page? maybe tooltipXCoordinate or something?

Reply:
its the local x coordinate within it's box